airflow/models/abstractoperator.py

```diff
@@ -284,13 +282,22 @@ def iter_mapped_dependants(self) -> Iterator[MappedOperator]:
         )
 
     def iter_mapped_task_groups(self) -> Iterator[MappedTaskGroup]:
-        """Return mapped task groups this task belongs to."""
+        """Return mapped task groups this task belongs to.
+
+        Groups are returned from the closest to the outmost.
+
+        :meta private:
```

Comment:
why meta private and not leading underscore?

Reply:
This is going to be called by XComArg in subsequent implementation.